### DISCUSSION TITLE
MQTT Settings

### DISCUSSION THREAD

**Discussion by yaleman** [dl9rdz/rdz_ttgo_sonde#271] [Q&A] [answered]
What do these do? Are they just a way of sending data about things it's seen somewhere else?

Thanks for this project, it's pretty amazing 😄

**Comment by dl9rdz** [comment, marked as answer]
Basically yes, just another alternative to APRS/AXUDP/SondeHub.
Has not been used much in the past (at least not that I was aware of, the code is also a contribution from someone else), but this seems to be changing now, there is increasing interest, and possibly there might be some changes in the MQTT payload format in the future.

**Reply by yaleman** [reply]
Thanks!
